### DISCUSSION TITLE
Bad Request (400 )

### DISCUSSION THREAD

**Discussion by moinulctg** [netbox-community/netbox#9337] [Q&A]
while i try to access netbox using http://127.0.0.1:8000 i am getting this error . i have installed netbox 3.1.7 in ubuntu virtualbox.Could anyone please help me on this

**Comment by markkuleinio** [comment]
Last time I got 400 Bad Request it was because I had ALLOWED_HOSTS configured incorrectly in NetBox configuration.py.

https://docs.netbox.dev/en/stable/configuration/required-settings/#allowed_hosts

**Reply by moinulctg** [reply]
ALLOWED_HOSTS = ['*'] , i configured like this . But same error

**Reply by markkuleinio** [reply]
And you restarted NetBox after the reconfiguration?

**Reply by moinulctg** [reply]
actually it was in my initial configuration . Can you please share , which guideline you have followed. Thanks for your reply

**Reply by markkuleinio** [reply]
I have followed https://docs.netbox.dev/en/stable/installation/

**Reply by moinulctg** [reply]
you have used centos or ubuntu . which netbox version you have installed?

**Reply by markkuleinio** [reply]
I've used Ubuntu, Debian, Amazon Linux (this I don't recommend ever), 3.1.x and 3.2.2

**Reply by moinulctg** [reply]
this I don't recommend ever - i didn't understand , what you mean by that

**Reply by markkuleinio** [reply]
Amazon Linux

**Reply by moinulctg** [reply]
ubuntu which version ? i have used 20.04 and netbox 3.1.7

**Reply by markkuleinio** [reply]
20.04 yes, I don't know why you are installing old NetBox

**Reply by moinulctg** [reply]
i have used version 3.2.2. but still same error

**Reply by moinulctg** [reply]
finally resolved after changing ALLOWED_HOSTS = ['*'] to ALLOWED_HOSTS = ['127.0.0.1'] or ALLOWED_HOSTS = ['Machine IP']
